doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -1,16 +1,54 @@
 [id='offline-repo-proc']
-= Preparing a Maven mirror repository for offline use 
+= Preparing a Maven mirror repository for offline use
 
 If your {OPENSHIFT} environment does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment.
 
 You do not need to complete this procedure if your {OPENSHIFT} environment is connected to the Internet.
```

Comment:
The RHPAM-1282 JIRA does not have any mention of openshift environment, cusomters that have on-premise setup might get confused by this.

Reply:
This particular document is for OpenShift. I have created https://issues.jboss.org/browse/BXMSDOC-4347 to make this procedure available without OpenShift.

Reply:
Great so please let me know when you have a PR for the on premise scenario that is valid for RHPAM-1282. I will get back to this once I  finish with CR1.

Reply:
It will not be done in time for 7.4GA but we will work for 7.4.1. Can you approve this one in the meantime?

Reply:
I can't approve. I didn't test it properly

Reply:
Time is of the essence so I'll have to merge, but any changes coming in this week or Monday July 1 will still be reflected in 7.4.

Reply:
As I said, I did not verify that this scenario works.

Reply:
@mramendi Do you have any JIRA that this openshift scenario is related to so I can get a better picture?

Reply:
No, I do not have a JIRA for this improvement except BXMSDOC-4209, which is the JIRA for making the documentation changes. This is simply an application of RHPAM-1282 to OpenShift. The scenario is the same, an environment without a connection to the public Internet; but the deployment is on OpenShift. 

In fact I took this on because I previously got field requests for this scenario and already had a limited scenario documented for 7.3.1 (the limitation was that it required existing service source). I ran a test environment on minishift to get the right Maven mirror settings.

I understand there is an on-premise need and have created a JIRA. It will now be easy as we can reuse the procedure for creating the mirror. We can document changing settings.xml to use a local mirror on-prem, which is quite doable, but, I think, even with the current extension we don;t have resources to do it for 7.4GA.